BrentOzarULTD/SQL-Server-First-Responder-Kit
# Enhancements #1388, #1396, #1389

Fixes #1388  
Fixes #1396
Fixes #1389

# Changes proposed in this pull request:
 - Changed view @OutputTableNamePerfmonStats_View to a more performing code for a big amount of data
 - Changed view @OutputTableNameFileStats_View in the same way as above
 - Added view @OutputTableNamePerfmonStatsActuals_View with pre-computed and usable values for perfmon counters 

# How to test this code:
With the current version installed, launch the update SP with output to table:

```
sp_BlitzFirst   @OutputDatabaseName = 'TestBlitz', 
  @OutputSchemaName = 'dbo', 
  @OutputTableName = 'BlitzFirst',
  @OutputTableNameFileStats = 'BlitzFirst_FileStats',
  @OutputTableNamePerfmonStats = 'BlitzFirst_PerfmonStats',
  @OutputTableNameWaitStats = 'BlitzFirst_WaitStats',
  @OutputTableNameBlitzCache = 'BlitzCache'
```
Even better if you already have existing data.

Backup the data in the views somewhere like this:

```
  SELECT * 
  INTO dbo.BlitzFirst_FileStats_Deltas_Original
  FROM dbo.BlitzFirst_FileStats_Deltas

  SELECT * 
  INTO dbo.BlitzFirst_PerfmonStats_Deltas_Original
  FROM dbo.BlitzFirst_PerfmonStats_Deltas
```

- Backup the tables defined above (rename them)
- Update the SP
- Delete the Views for PerfmonStats and FileStats Deltas
- Launch the SP again as above
- The views are now re-created
- Delete the new tables and restore the original ones
- The data in the updated BlitzFirst_PerfmonStats_Deltas and BlitzFirst_FileStats_Deltas views should match the backed up data from the views saved before.

-----
Testing "@OutputTableNamePerfmonStatsActuals_View" is more difficult since it's a new feature
I have on the pipeline a modification to the PowerBI model that uses this view, in that way it will be easier to see if the data makes sense.


Has been tested on (remove any that don't apply):
 - SQL Server 2014
 - SQL Server 2016
  - SQL Server 2017
